.github/workflows/installcheck.yml

```diff
@@ -0,0 +1,45 @@
+name: Regression
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get latest commit id of PostgreSQL 11
+        run: |
+          echo "PG_COMMIT_HASH=$(git ls-remote git://git.postgresql.org/git/postgresql.git refs/heads/REL_11_STABLE | awk '{print $1}')" >> $GITHUB_ENV
+
+      - name: Cache PostgreSQL 11
+        uses: actions/cache@v2
+        id: pg11cache
+        with:
+          path: ~/pg11
+          key: ${{ runner.os }}-v1-pg11-${{ env.PG_COMMIT_HASH }}
+
+      - name: Install PostgreSQL 11
+        if: steps.pg11cache.outputs.cache-hit != 'true'
+        run: |
+          git clone --depth 1 --branch REL_11_STABLE git://git.postgresql.org/git/postgresql.git ~/pg11source
+          cd ~/pg11source
+          ./configure --prefix=$HOME/pg11 CFLAGS="-std=gnu89"
```

Comment:
Ansiflag can be given, but the comment in the age contains a single comment that is not compatible with ansi(`-ansi`), so specify it as std98(`-std=gnu98`).